src/devices/xiaomi.ts

```diff
@@ -2558,7 +2654,80 @@ const definitions: Definition[] = [
         exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
             e.power(), e.current(), e.energy(), e.voltage(), e.device_temperature(),
         ],
-        extend: [xiaomi.extend.switchType],
+        extend: [
+            xiaomi.extend.switchType,
+            extend.binary({
+                name: 'interlock',
+                valueOn: true,
+                valueOff: false,
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x02d0, type: 0x10},
+                description: 'Enabling prevents both relays being on at the same time (Interlock)',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
+                name: 'power_on_behavior',
+                lookup: {'on': 0, 'previous': 1, 'off': 2, 'reverse': 3},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0517, type: 0x20},
+                description: 'Controls the behavior when the device is powered on after power loss',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
```

Comment:
This can be moved to xiamo lib and be made a function such that you can do the following here:

```js
xiaomi.extend.operationMode('l1'),
xiaomi.extend.operationMode('l2'),

```